docs/src/rules/no-constant-binary-expression.md

```diff
@@ -0,0 +1,68 @@
+# no-constant-binary-expression
+
```

Comment:
```suggestion

Disallows expressions where the operation doesn't affect the value.

```

We usually add the short description (basically the same text as `meta.docs.description`) as the first paragraph in the docs.